codalab/worker/bundle_manager.py

```diff
@@ -529,3 +533,20 @@ def _construct_run_message(self, worker, bundle):
         resources['request_network'] = bundle.metadata.request_network
 
         return message
+
+    def _bring_zombie_bundles_to_fail(self):
+        failure_message = "Bundle exceeds maximum time limit {}.".format(BUNDLE_TIMEOUT_SECONDS)
+        bundles_to_fail = (
+            self._model.batch_get_bundles(state=State.UPLOADING)
```

Comment:
Is there an easy way to make one database call here?  If not, okay to leave.

Reply:
I tried to put all three conditions in same call, but it complains about repeated keyword in this function: https://github.com/codalab/codalab-worksheets/blob/master/codalab/model/bundle_model.py#L126. It takes a dictionary.

Reply:
Ok, maybe okay to leave it as is.